mailpoet/assets/js/src/automation/editor/components/workflow/index.tsx

```diff
@@ -4,6 +4,7 @@ import {
   __unstableUseCompositeState as useCompositeState,
 } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
+import { EditorNotices } from '@wordpress/editor';
```

Comment:
@websupporter Ahh, I just needed to use notices and run into this. We're not using the `@wordpress/editor` package and therefore it should never be imported anywhere in the automation code! I wonder if should [add some lints for this](https://eslint.org/docs/latest/rules/no-restricted-imports) but maybe it's not worth it because extracting it to a package within the monorepo with its deps will solve it.

I fixed it in my branch: https://github.com/mailpoet/mailpoet/commit/ad3a3498bd5cb46f26d4fa6f78989b4b469397d7

(This was also the reason why the notices looked different than in WP.)